app/AppContainer.js

```diff
@@ -65,6 +66,10 @@ export class AppContainer extends Component {
                 return viewport(loadMask({isDisplayed: true}));
             case LoadState.LOGIN_REQUIRED:
                 return loginPanel();
+            case LoadState.UNAUTHORIZED:
```

Comment:
Let's call this 'ACCESS_DENIED' to link up to the the 'XH.accessDeniedMessage'  (or vice-versa, just need to be consistent.)